_episodes/06-raster-intro.md

```diff
@@ -0,0 +1,462 @@
+---
+title: "Read and visualize raster data"
+teaching: 40
+exercises: 20
+questions:
+-  "What is a raster dataset?"
+-  "How do I read and plot raster data in Python?"
+-  "How can I handle missing values for a raster?"
+
+objectives:
+-  "Describe the fundamental attributes of a raster dataset."
+-  "Explore raster attributes and metadata using Python."
+-  "Read rasters into Python using the `rioxarray` package."
+-  "Visualize single-/multi- band raster data."
+
+keypoints:
+- "One can use `rioxarray.open_rasterio()` to read raster data"
+- "`rioxarray` stores CRS information as a CRS object that can be converted to an EPSG code or PROJ4 string."
+- "It is good to consider using nan to represent no data value(s), for statistics and visualization. "
```

Comment:
```suggestion
- "Consider using `NaN` to represent no data value(s) for statistics and visualization."
```

Reply:
@fnattino I think we should indeed suggest using NaNs to encode the missing value? Since it can be handled well by most visualization/statistics packages?

Reply:
I eventually changed into this:

```
- "`rioxarray` and `xarray` are for working with multidimensional arrays like pandas is for working with tabular data."
- "`rioxarray` stores CRS information as a CRS object that can be converted to an EPSG code or PROJ4 string."
- "Missing raster data are filled with nodata values, which should be handled with care for statistics and visualization."
```